src/mixins/richEditor/index.js

```diff
@@ -104,6 +104,10 @@ export default {
 		 * @return {string}
 		 */
 		genSelectTemplate(value) {
+			if (typeof value === 'undefined') {
+				return `${this.autocompleteTribute.current.collection.trigger}${this.autocompleteTribute.current.mentionText}`
```

Comment:
Can you add a comment saying something like:
> // What's returned by this function will replace the trigger '@' and the search text, so when there is no match, then we simply return '@' and the text

Reply:
Done :-)